python/sglang/srt/layers/moe/topk.py

```diff
@@ -128,8 +128,7 @@ def grouped_topk(
     return topk_weights.to(torch.float32), topk_ids.to(torch.int32)
 
 
-@torch.compile(dynamic=True, backend=get_compiler_backend())
```

Comment:
I think the Python-related part should be updated after sgl-kernel releases a new version. Can you split it into two PRs? Thank you.

Reply:
Of course! 🫡